authentication.md

```diff
@@ -120,4 +122,21 @@ Embedded
 - Embeds the authenticated eperson
 
 Return code
-- 200 Ok in all the scenario both authenticated than not authenticated (valid token, invalid token or missing token)
\ No newline at end of file
+- 200 Ok in all the scenario both authenticated than not authenticated (valid token, invalid token or missing token)
+
+## Log in as
+
+For any request, an `x-on-behalf-of` header can be included.
```

Comment:
just a style note I would prefer the use of X-On-Behalf-Of as used in the SWORD protocol spec but this is just "layout" as header by spec are case insensitive